testing/e2e/expect.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {ControllerPromise} from './functional-test-controller';
+import chai, {expect as chaiExpect} from 'chai';
```

Comment:
In addition to `chai`, we use `chai-as-promised` in our unit tests do run various kinds of assertions. The last I checked, we couldn't use it in integration tests because it was incompatible with Sauce Labs browsers. See comments below:
https://github.com/ampproject/amphtml/blob/63348d5126a69dc1fffa60bfde41a93af231faee/test/chai-as-promised/chai-as-promised.js#L17-L36

It might be worth seeing if that restriction still holds, and if not, making `chai-as-promised` available to these tests.

Reply:
These tests use a different promise strategy than `chai-as-promised`, since these use async-await, so I don't think it would make sense to make the package available to these tests.